gui/wxpython/lmgr/layertree.py

```diff
@@ -1189,6 +1216,49 @@ def OnPopupProperties(self, event):
         """Popup properties dialog"""
         self.PropertiesDialog(self.layer_selected)
 
+    def OnPopupGroupOpacityLevel(self, event):
+        """Popup opacity level indicator for group of layers"""
+        # Get opacity level from the first finded map layer
+        child, cookie = self.GetFirstChild(self.layer_selected)
+        while child:
+            maplayer = self.GetLayerInfo(child, key="maplayer")
+            ltype = self.GetLayerInfo(child, key="type")
+            if maplayer and ltype != "command":
+                break
+            child = self.GetNextSibling(child)
+            if child is None:
+                child, cookie = self.GetNextChild(child, cookie)
+        current_opacity = maplayer.GetOpacity()
+        dlg = SetOpacityDialog(
+            self,
+            opacity=current_opacity,
+            title=_("Set opacity of <{}>").format(maplayer.GetName()),
```

Comment:
Could this have the name of the group instead? Right now it shows the name of the first layer.

Reply:
Fixed with https://github.com/OSGeo/grass/pull/3583/commits/61c390e19ee2dbaeb9b678eacd1ae454b72a00d3.